financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -98,10 +100,19 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity() {
         setContent {
             FinancialConnectionsTheme {
                 val state by viewModel.stateFlow.collectAsState()
-                NavHost(
-                    initialPane = state.initialPane,
-                    testMode = state.testMode,
+                val bottomSheetState = rememberStripeBottomSheetState(
+                    initialValue = ModalBottomSheetValue.Expanded,
```

Comment:
We launch this expanded, because `FinancialConnectionsSheetActivity` is already open.